source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
```

Comment:
Doesn’t Azure already define these environment variables? (@pmeredit?)

If so, I feel like we shouldn’t tell users to define them; rather, say that Azure usually sets them. We might even be able just to link to external documentation?

Reply:
It should say they need to be set.